bin/compile

```diff
@@ -30,7 +30,13 @@ curl --silent --retry 5 $BASE_URL/$SIGNATURE --output $SIGNATURE_PATH
 curl --silent --retry 5 $BASE_URL/$CHECKSUMS --output $CHECKSUMS_PATH
 
 echo "Verifying download" | indent
-gpg --quiet --keyserver pgp.mit.edu --recv 7BFB4EDA || gpg --quiet --keyserver keys.gnupg.net --recv 7BFB4EDA
+
+for i in {1..4}; do
+    gpg --quiet --keyserver keys.gnupg.net --recv 7BFB4EDA
+    if [ $? = 0 ]; then
+	break;
+    fi
+done
```

Comment:
Please don't mix spaces and tabs, and please follow the existing indent style in this file (two spaces).

<img width="427" alt="screen shot 2018-07-26 at 22 41 23" src="https://user-images.githubusercontent.com/42626/43287340-18ccae4e-9125-11e8-95bc-3fa0ab3d30a3.png">

Reply:
Yes, will clean up the whitespace.